docs/src/markdown/setup/Setup-Mixed-Docker.md

```diff
@@ -27,6 +27,7 @@ For this to work, your Aegeus wallet needs to bind to an external IP
     rpcport=51473
     rpcconnect=192.168.178.20
     rpcallowip=192.168.178.20
+    printtoconsole=1
```

Comment:
If you change the markdown you'd need to regenerate the docs and change the main README. Should this option not be the default for the aegd? If so, please create an issue about that in [AegeusCoin/aegeus](https://github.com/AegeusCoin/aegeus/issues)